packages/bpk-docs/src/layouts/SideNavLayout/nav-list.scss

```diff
@@ -65,23 +66,26 @@
     }
 
     &--active {
-      background-color: $bpk-color-blue-500;
       color: $bpk-color-white;
       font-weight: bold;
       text-decoration: none;
 
+      @include bpk-themeable-property(background-color, --bpk-link-active-color, $bpk-color-blue-500);
+
       @include bpk-hover {
-        background-color: $bpk-color-blue-500;
         color: $bpk-color-white;
         font-weight: bold;
         text-decoration: none;
+
+        @include bpk-themeable-property(background-color, --bpk-link-hover-color, $bpk-color-blue-500);
```

Comment:
This shouldn't change color so it should be the active color too.

Reply:
Good point - now updated.